neutrino.go

```diff
@@ -999,7 +999,7 @@ func (s *ChainService) rollBackToHeight(height uint32) (*waddrmgr.BlockStamp, er
 	}
 
 	for uint32(bs.Height) > height {
-		header, _, err := s.BlockHeaders.FetchHeader(&bs.Hash)
+		header, headerHeight, err := s.BlockHeaders.FetchHeader(&bs.Hash)
```

Comment:
🤔

Reply:
So block disconnects haven't been delivering the proper information this entire time? Wouldn't this also then propagate to `lnd` where we exercise these scenarios in the chain notifier?

Reply:
Yeah, this could be the cause to some of the neutrino reorg tests within lnd failing as they depend on heights, though I haven't narrowed anything down.

Reply:
IIRC, we ended up disabling one or two of them, so we can note to enable them again once this goes in and `lnd` itself is updated.